rake/build/environment.rb

```diff
@@ -81,8 +81,8 @@
 pipeopts 'st2mistral' do
   checkout true
   envpass :giturl,  'https://github.com/StackStorm/mistral', from: 'ST2MISTRAL_GITURL'
-  envpass :gitrev,  'st2-2.1.0',                             from: 'ST2MISTRAL_GITREV'
+  envpass :gitrev,  'master',                                from: 'ST2MISTRAL_GITREV'
   envpass :gitdir,  make_tmpname('mistral-')
-  envpass :mistral_version, '2.1.0'
+  envpass :mistral_version, '2.2dev'
```

Comment:
Defaults generated wrong mistral package from old version: 
```sh
ls /tmp/st2-packages/
st2-2.2dev-1.x86_64.rpm  st2mistral-2.1.0-1.x86_64.rpm
```

Changed to reflect current settings for `master`.

When running build via `CircleCI` we populate them this way, see: 
- https://github.com/StackStorm/st2-packages/blob/master/.circle/buildenv_mistral.sh#L32-L34
- https://github.com/StackStorm/st2-packages/blob/master/.circle/buildenv_mistral.sh#L17
- https://github.com/StackStorm/mistral/blob/master/version_st2.py